src/rules/selector-type-no-unknown/__tests__/index.js

```diff
@@ -62,6 +62,9 @@ testRule(rule, {
   }, {
     code: "@include keyframes(identifier) { 0% {} 100% {} }",
     description: "non-standard usage of keyframe selectors",
+  }, {
+    code: "a { --custom-property-set: {} }",
```

Comment:
I think it's supposed to have a semicolon after the `{}`, so `a { --custom-property-set: {}; }`